virtcontainers/device/config/config.go

```diff
@@ -205,6 +215,10 @@ type VhostUserDeviceAttrs struct {
 	Tag       string
 	CacheSize uint32
 	Cache     string
+
+	// PCIAddr is the PCI address used to identify the slot at which the drive is attached.
+	// It is only meaningful for vhost user devices
+	PCIAddr string
```

Comment:
Wondering if we should rename the existing `MacAddress` string in this to `Addr` and use that to store the PCIAddr in case of vhost-usr-block, mac address in case of vhost-user-net and in case of (still to be handled )vhost-user-scsi devices maybe use that to store SCSI address if required.
Would like to hear what others think about this.
cc @jodh-intel

Reply:
sounds good to me.
And could it be revised in some later PR to remove duplicated elements if jodh also agrees with us? :)

Reply:
Got it, and just added a new issue #2433 as a reminder.